packages/server/src/app/models/user_quiz/user_quiz.entity.js

```diff
@@ -0,0 +1,50 @@
+const { DataTypes, Model } = require('sequelize')
```

Comment:
Make unique constraint:
https://stackoverflow.com/questions/42195348/how-to-define-unique-index-on-multiple-columns-in-sequelize

Reply:
Is this something that still needs doing?